src/cli/explorer.rs

```diff
@@ -48,12 +48,6 @@ pub async fn explorer(opts: &ExplorerOpts) {
         let _ = kill_container(CONTAINER_NAME).await;
     }
 
-    run_docker_image(
-        "ghcr.io/lambdaclass/stark_compass_explorer:v0.2.34.3-1",
-        CONTAINER_NAME,
-        Some(env),
-        Some(host_config),
-    )
-    .await;
+    run_docker_image("anshalshukla/starkcompass:latest", CONTAINER_NAME, Some(env), Some(host_config)).await;
```

Comment:
let's ask them to create a release? or let's push it on our ECR if you think that will take time